scene/gui/dialogs.cpp

```diff
@@ -41,12 +42,34 @@
 #include "scene/main/window.h" // Only used to check for more modals when dimming the editor.
 #endif
 
-// AcceptDialog
-
 void AcceptDialog::_input_from_window(const Ref<InputEvent> &p_event) {
 	Ref<InputEventKey> key = p_event;
 	if (key.is_valid() && key->is_pressed() && key->get_keycode() == KEY_ESCAPE) {
 		_cancel_pressed();
+		return;
+	}
+
+	Ref<InputEventMouseMotion> mm = p_event;
+	if (mm.is_valid() && dragging) {
+		DisplayServer::WindowID id = get_window_id();
+		Point2 p = DisplayServer::get_singleton()->window_get_position(id);
+		DisplayServer::get_singleton()->window_set_position(p + mm->get_relative(), id);
```

Comment:
This won't work consistently on most OSes, and will result in windows lagging behind the mouse or jumping around as long as it's moved for more than a fraction of the second.

Use `DisplayServer::get_singleton()->mouse_get_absolute_position()` to get mouse position that is independent of the current window (position in screen coordinates).